app/kumactl/cmd/inspect/inspect_dataplanes.go

```diff
@@ -23,6 +23,7 @@ type inspectDataplanesContext struct {
 	tagsArgs struct {
 		tags map[string]string
 	}
+	isGateWayDataplanes bool
```

Comment:
Rename `tagsArgs` to `args`.
Rename `isGateWayDataplanes` to `gateway` and include it into `args` struct

Reply:
done